src/overlay.jsx

```diff
@@ -133,7 +95,9 @@ const Overlay = React.createClass({
     const styles = getStyles(this.props, this.state);
 
     return (
-      <div {...other} ref="overlay" style={prepareStyles(Object.assign(styles.root, style))} />
+      <div {...other} ref="overlay" style={prepareStyles(Object.assign(styles.root, style))}>
+        {this.props.autoLockScrolling && <AutoLockScrolling lock={this.props.show} />}
```

Comment:
autoLockScrolling is here to disable the behavior. (no idea why).

It's not used anywhere. I'am gonna remove it, cannot think why it would be useful.

Tell me if you want it back.